<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>regi</class>
 <widget class="QWidget" name="regi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-image: url(:/Desktop/IMG_9514.JPG);
    background-repeat: no-repeat;
    background-size: cover;
}
QGroupBox {
    background: transparent;
    border: 2px solid gray;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/database/576.webp);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size:20px}</string>
      </property>
      <property name="text">
       <string>Admin Registration</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size:20px}</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size:20px}</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
font-size:20px}</string>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>220</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-size:20px}</string>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>260</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-size:20px}</string>
      </property>
      <property name="text">
       <string>Login Page</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
